### PR TITLE
TC_01.004.01 | Enable or disable project

### PR DESCRIPTION
**Pull request by FilipRahulia**
https://trello.com/c/G2ycVvCh/226-tc0100401-freestyle-project-enable-or-disable-project

### CODE REVIEW

**Comment by judmi** [comment]
Link to the trello card? :)

**Comment by FilipRahulia** [comment]
> Link to the trello card? :)

thx, added

File: src/test/java/school/redrover/FreestyleProject3Test.java [hunk 1]
```diff
@@ -9,22 +9,30 @@
...
         getDriver().findElement(By.xpath("//a[@href = '/view/all/newJob']")).click();
-        getDriver().findElement(By.id("name")).sendKeys(projectName);
+        getDriver().findElement(By.id("name")).sendKeys(PROJECT_NAME);
         getDriver().findElement(By.className("hudson_model_FreeStyleProject")).click();
         getDriver().findElement(By.id("ok-button")).click();
         getDriver().findElement(By.name("Submit")).click();
         getDriver().findElement(By.id("jenkins-home-link")).click();
+    }
+
+    public void createAndOpenFreestyleProject() {
```
Comment (EvgenyJavaTest) [inline src/test/java/school/redrover/FreestyleProject3Test.java:22]: This method almost completely duplicates createFreestyleProject() - 6 out of 7 lines match.

Comment (FilipRahulia) [reply src/test/java/school/redrover/FreestyleProject3Test.java:22]: fixed

**Review by SafronovaOlga — APPROVED** [review]

**Review by verafes — APPROVED** [review]

**Pull request merged**